plugin/pymode.vim

```diff
@@ -106,7 +106,7 @@ call pymode#default("g:pymode_lint_on_fly", 0)
 " Show message about error in command line
 call pymode#default("g:pymode_lint_message", 1)
 
-" Choices are: pylint, pyflakes, pep8, mccabe
+" Choices are: pylint, pyflakes, pep8, mccabe and pep257
```

Comment:
@fmv1992, are there other linters to be added here? Such as `pycodestyle`, `pydocstyle`, etc?